ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -140,8 +146,17 @@
   :ssl?                 - allow connections over HTTPS
   :ssl-port             - the SSL port to listen on (defaults to 443, implies
                           :ssl? is true)
-  :exclude-ciphers      - When :ssl? is true, exclude these cipher suites
-  :exclude-protocols    - When :ssl? is true, exclude these protocols
+  :exclude-ciphers          - When :ssl? is true, additionally exclude these
+                              cipher suites
+  :replace-exclude-ciphers? - use the value of :exclude-ciphers to replace
+                              the cipher exclusion list rather than adding to it
+                              (defaults to false)
+  :exclude-protocols          - When :ssl? is true, additionally exclude these
+                                protocols
+  :replace-exclude-protocols? - use the value of :exclude-protocols to replace
+                                the protocols exclusion list rather than adding
+                                to it
+                                (defaults to false)
```

Comment:
What about formatting/writing the docstring like so:

```
  :ssl-port             - the SSL port to listen on (defaults to 443, implies
                          :ssl? is true)
  :exclude-ciphers      - when :ssl? is true, additionally exclude these
                          cipher suites
  :exclude-protocols    - when :ssl? is true, additionally exclude these
                          protocols
  :replace-exclude-ciphers?   - when true, :exclude-ciphers will replace rather
                                than add to the cipher exclusion list (defaults
                                to false)
  :replace-exclude-protocols? - when true, :exclude-protocols will replace
                                rather than add to the protocols exclusion list
                                (defaults to false)
```

Reply:
updated